tools/biosnoop.bt

```diff
@@ -20,26 +20,31 @@ tracepoint:block:block_io_start
 	$key = (args.dev, args.sector);
 	@start[$key] = nsecs;
 	@iopid[$key] = pid;
+	@iocomm[$key] = comm;
 }
 
 tracepoint:block:block_io_done
-/@start[args.dev, args.sector] != 0 && @iopid[args.dev, args.sector] != 0 && args.comm != ""/
+/@start[args.dev, args.sector] != 0 &&
+ @iopid[args.dev, args.sector] != 0 &&
+ @iocomm[args.dev, args.sector] != ""/
```

Comment:
supernit while you're here, can we use the `has_key` func for these two?

Reply:
Good idea, will do.

Reply:
Or, you know what, let's just do it in a separate PR (which doesn't have to go to the release branch) as `has_key` can be used in a bunch of other tools.

Reply:
Sure - sounds good.